### DISCUSSION TITLE
Add a field to download the data set (downloadURL)

### DISCUSSION THREAD

**Discussion by nichtich** [datacite/datacite-suggestions#132] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

Its not possible to automatically get the actual data publication described by a DataCite record.

### What solution might meet your needs?

Add an optional field `downloadURL`  for instance

~~~xml
<identifier identifierType="DOI">10.5281/zenodo.10569321</identifier>
<downloadURL>https://zenodo.org/api/records/10569321/files-archive</downloadURL>
~~~

or 

~~~xml
<identifier identifierType="DOI">10.5281/zenodo.10569321</identifier>
<downloadURLs>
  <downloadURL type="full">https://zenodo.org/api/records/10569321/files-archive</downloadURL>
  <downloadURL type="file" path="README.md">https://zenodo.org/records/10569321/files/README.md?download=1</downloadURL>
...
~~~

### Your name

Jakob Voß

### Your organization

VZG

### What alternatives have you tried or considered?

Try to automatically find out the base URL and software of the repository (e.g. Zenodo with InvenioRDM) and write an extractor script to access its individual API. This is a hack and does not scale.

### Is there anything else you would like to share?

Our use case is an application where people can submit a DOI to download and analyze the publication data.

The  field can be mapped to [dcat:downloadURL](https://www.w3.org/TR/vocab-dcat-2/#Property:distribution_download_url)

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [X] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [X] Individuals or institutions analyzing DataCite DOIs and metadata
- [X] DataCite staff
- [ ] Other

**Comment by adambuttrick** [comment]
@nichtich While separate from DataCite functionality, I've had good luck using [DataHugger](https://github.com/J535D165/datahugger) for this purpose.

**Comment by KellyStathis** [comment]
We’re looking to gather more info on this topic! I’ve updated this related discussion: [Add "distribution" information for direct file links #42](https://github.com/datacite/datacite-suggestions/discussions/42) including a link to a short survey.  Let us know if you have any questions.

**Comment by KellyStathis** [comment]
We've shared a new proposal for a "Distribution" property here: https://github.com/datacite/datacite-suggestions/discussions/211 which contains a contentURL (similar to downloadURL).
